tests/suites/actions/run_actions.sh

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Canonical Ltd.
+# Licensed under the AGPLv3, see LICENCE file for details.
+
+run_actions_params() {
+	echo
+	model_name="test-actions"
+	log_file="${TEST_DIR}/${model_name}.log"
+
+	ensure "juju-qa-action" "${log_file}"
+
+	juju deploy juju-qa-action
+
+	wait_for "juju-qa-action" "$(idle_condition "juju-qa-action")"
+
+	# Run a valid action.
+	juju run juju-qa-action/0 fortune length="long"
+	# Check the the task succeeded.
+	juju show-operation 1 --format=json | jq '.status' | check 'completed'
+	# Check the action succeeded.
+	juju show-task 2 --format=json | jq '.status' | check 'completed'
+
+	# Run an action that will return failed status.
+	juju run juju-qa-action/0 fortune length="long" fail="fail with this string"
+	# Check the the task succeeded.
+	juju show-operation 3 --format=json | jq '.status' | check 'failed'
```

Comment:
The comment is wrong, as the check failed (task failed successfully?)

Reply:
Changed